hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -419,9 +419,7 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
     }
 
     loginOMUserIfSecurityEnabled(conf);
-
-    this.allowListAllVolumes = conf.getBoolean(OZONE_OM_VOLUME_LISTALL_ALLOWED,
-        OZONE_OM_VOLUME_LISTALL_ALLOWED_DEFAULT);
+    setInstanceVariablesFromConf();
```

Comment:
Minor: Similarly do we need  to remove isAclEnabled from OzoneManager L440

Reply:
Ah, you are correct. Well spotted. I have fixed this and pushed another commit.